prometheus-config.sh

```diff
@@ -12,8 +16,15 @@ for arg; do
         (--compose) COMPOSE=1
             AM_ADDRESS="aalert:9093"
             ;;
-        (--no-cas-cdc) COMPOSE=1
-            NO_CAS_CDC="1"
+        (--no-cas-cdc)
```

Comment:
This is undocumented?

Reply:
on purpose, it's there just for the cloud and for backwards compatiblity